metrics/metrics.go

```diff
@@ -26,6 +26,34 @@ type AdapterLabels struct {
 	AdapterErrors map[AdapterError]struct{}
 }
 
+// AdapterOverheadType: adapter overhead type enumeration
+type AdapterOverheadType string
+
+const (
+	PreBidderRequest   AdapterOverheadType = "pre-bidder-request"
+	PostBidderResponse AdapterOverheadType = "post-bidder-request"
```

Comment:
Nitpick: I'm not even sure if the team will agree on this. The naming/wording for both `"pre-bidder-request"` and `"post-bidder-request"` makes a lot of sense to me, except that it doesn't reflect the fact that this metric is only timing the successful scenarios. Given the fact that neither of these metrics are logged when an error is found, I think we should rename the metric names and labels to make that absolutely clear.  Maybe something in the ballpark of:
```
	NonErrorPreBidderOverhead  AdapterOverheadType = "non-error-pre-bidder-time"
	NonErrorPostBidderOverhead  AdapterOverheadType = "non-error-post-bidder-time"
```
I realize I don't always come up with the best variable names but, you get the point. I'm interested in your thoughts. Do you think this is even necessary? Should we modify these names to make more emphais to the fact that error scenarios are not accounted for?

Reply:
updated var names and added some comment to indicate its usage in https://github.com/prebid/prebid-server/pull/2658/commits/126fb22830afcf8e77acf7b2c1f680ea02778982